nursery/get-workstation-config.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: get workstation config
+    namespace: network/workstation
+    authors:
+      - kevross33/Kevin Ross
+    scopes:
+      static: function
+      dynamic: span of calls
+    att&ck:
+      - Discovery::System Network Configuration Discovery [T1016]
+    examples:
+      - f5fca1b178af87bd48c7ea9e3f2c957b
+  features:
+    - and:
+      - string: /net/i
+      - string: /config/i
+      - string: /workstation/i
```

Comment:
This is a little broad - is there anything else we could add to reduce the risk of FPs (e.g. `api`, `match` features)?